### DISCUSSION TITLE
Hope to join s905y2 4g dtb support

### DISCUSSION THREAD

**Discussion by quesur** [devmfc/debian-on-amlogic#52] [Q&A]
Armbian officially supports Radxa Zero's s905y2 SOC, but tvbox's s905y2 cannot be found. I have tried using Radxa Zero's DTB but it did not work
Hope to join s905y2 4g dtb support
🙏🙏🙏🙏🙏🙏🙏🙏🙏🙏🙏🙏

**Comment by devmfc** [comment]
AFAIK Radxa only offers vendor kernels.  
My images use mainline kernels.  
  
Did you try the t95w2_s905y2 box config?What tvbox do you have?

**Reply by quesur** [reply]
There is only t95w2s905y4 in the bag, but t95w2s905y2 was not found，The box is A95X, with no Ethernet port but 4GB of memory. I want to use an external USB network card. I have tried the DTB inside the package multiple times and found that s905x2_generic is working!, According to the research, it was found that s905y2 and s905x2 are very similar, but the memory is only recognized as 3267MB. The board has 4GB of memory. If 4GB of memory support can be added, that's great. In addition, WiFi Bluetooth is MT7668RSN and does not work. If it is supported, it would be even better. If it is not supported, it doesn't matter. The most important thing is to hope that the memory can support 4GB，Thank you for your reply🙏

**Reply by devmfc** [reply]
Argh... I was still sleeping when I gave that answer. I read s905y4, which is an S4. But you mentioned s905y2, which is, like you already figured, a G12A.  Sorry...

That you only see 3.2GiB is normal... I have never seen an image that can use the full 4GiB, and I think it is impossible, because a part of the address space is used by SOC Registers and reserved memory.

You could also try the a95xf2 box config, which also uses an MT7668 WiFi module. I never got Bluetooth to work on that module.

**Reply by quesur** [reply]
Okay, let me try A95xf2. Thanks
